controllers/event/index.js

```diff
@@ -325,3 +340,54 @@ export const getUserLocationEvents = async (req, res) => {
     data: eventsInUsersLocation
   });
 };
+export const updateEventStatus = async (req, res) => {
+  let token = req.headers.authorization;
```

Comment:
Are you using this code? I thought you changed the implementation

Reply:
Thaks I worked on this